### DISCUSSION TITLE
Add Appium as a Pre-installed application

### DISCUSSION THREAD

**Discussion by dmcgill50** [actions/virtual-environments#3726] [Installed software]
As a QA Engineer, it would be nice to have a hosted runner designed specifically for Appium based automated testing.

**Comment by AlenaSviridenko** [comment]
Hi @dmcgill50,
have you tried using action for setting up Appium? https://github.com/marketplace/actions/appium-server-action looks pretty popular. We do recommend using actions as they provide more flexibility https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#using-preinstalled-software.

If it doesn't work for you, please, create a feature request in our repository, and we will review it.
